src/js/history.js

```diff
@@ -3,31 +3,38 @@ const colArray = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H'];
 
 
 function getHistory() {
+    trackMoves.historyInAlgebraicNotation = [];
     for (let index = 0; index < trackMoves.history.length; index++) {
-        const currentMove = trackMoves.history[index]; // this one is dangling
 
         makeAlgebraicNotation(index);
     }
 }
 
 function makeAlgebraicNotation(index) {
-    let startRow = trackMoves.history[index].from.row;
-    let startCol = trackMoves.history[index].from.col;
-    let endRow = trackMoves.history[index].to.row;
-    let endCol = trackMoves.history[index].to.col;
+    const currentMove = trackMoves.history[index];
+    const moveNumber = index + 1;
+    const pieceMoved = currentMove.pieceSymbol;
```

Comment:
This line might not work after we renamed from "pieceSymbol" to "symbol".

@jorgen7c  Maybe you can fix and send another pull request. :)